app/frontend/helpers/printConversionRates.ts

```diff
@@ -4,7 +4,7 @@ const addThousandsCommas = (amount, fractionDigits) =>
     maximumFractionDigits: fractionDigits,
   })
 
-const formatFiat = (amount, maxDigits) => {
+const formatFiat = (amount, maxDigits=10) => {
```

Comment:
Find out where (and if all) we used maxDigits and what should be correct default!